app/src/components/HostFormDetail/HostFormInfo.tsx

```diff
@@ -224,7 +224,11 @@ const HostFormInfo: React.FC = () => {
                                     <Button
                                         variant="contained"
                                         color="primary"
-                                        style={{ margin: '0 3rem 0 7rem' }}
+                                        style={{
```

Comment:
In the near future these styling concerns should be informed by the theme provider, or at least factored into a local definition rather than inlined